### PR TITLE
updates upcoming events

### PR DESCRIPTION
**Pull request by minaorangina**

### CODE REVIEW

File: london.html [hunk 1]
```diff
@@ -120,9 +167,6 @@ <h5 class="header time">
             </div>
           </div>
         </div>
-
-        <!--   Past Events Section   -->
-        <h3 class="header center green-text">Past events</h3>
```
Comment (claireinez) [inline london.html:125]: Why did you get rid of the "Past events" header?

File: events.html [hunk 2]
```diff
@@ -72,6 +72,55 @@ <h3 class="header center green-text">Upcoming Events</h3>
 
       <div class="event-container">
         <a href="london.html"><div class="chip">London</div></a>
+        <h5 class="header light event-container__header">September 2017</h5>
```
Comment (iteles) [inline events.html:75]: :tada:

File: london.html (hunk 1 above)
Comment (minaorangina) [reply london.html:125]: Accidental!  Good spot

Comment (minaorangina) [reply london.html:125]: Actually upon closer inspection it's still there, think it's just a weird diff

**Review by msmichellegar — APPROVED** [review]
:)

File: london.html (hunk 1 above)
Comment (claireinez) [reply london.html:125]: Ha you're correct! Didn't spot it had moved!

**Review by claireinez — APPROVED** [review]

**Pull request merged**